app/schema.py

```diff
@@ -24,7 +24,31 @@ class Post(BaseModel):
     desc: str
     long: Decimal
     lat: Decimal
-    images: List[Images]
+    # images: List[Images]
+
+    class Config:
+        orm_mode = True
+
+
+class PostCreate(BaseModel):
+    title: str
+    type: PostType
+    desc: str
+    long: Decimal
+    lat: Decimal
+    # images: List[Images]
```

Comment:
Wondering: couldn't these Post classes be inherited?

Reply:
because i need to set some fields to be not requested.

Reply:
I see! Closing.